errutil/rt_error.go

```diff
@@ -1,6 +1,12 @@
 package errutil
 
-import "net/http"
+import (
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+var Rt_err = errors.New("RoundTripper error")
```

Comment:
My suggestion would be to hide this variable and create a function for it:

```suggestion
var rtErr = errors.New("RoundTripper error")

func IsMockedError(err error) bool {
  return errors.Is(err, rtErr)
}
```

This is mostly because `Rt_err` isn't a constant (and it cannot be). This will make it more maintainable IMO